matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/RoomSummaryQueryParams.kt

```diff
@@ -86,7 +86,7 @@ data class RoomSummaryQueryParams(
         /**
          * Used to filter room using the current space.
          */
-        val spaceFilter: SpaceFilter?,
+        val spaceFilter: SpaceFilter,
```

Comment:
Maybe mention this API change in a file `6666.sdk`

Reply:
https://github.com/vector-im/element-android/pull/6666/commits/e4640f14d24529ba0efe1e025d09d37714f5a712